### ISSUE TITLE
Trigger autocompletion Ubuntu/Arch

### ISSUE THREAD

**Issue by pcoves** [neoclide/coc.nvim#1542]
### Ubuntu

```
## versions

vim version: VIM - Vi IMproved 8.0 (2016 Sep 12, compiled Jun 06 2019 17:31:41)
node version: v8.10.0
coc.nvim version: 0.0.74-98a709ec3f
term: dumb
platform: linux
```

### ArchLinux

```
## versions

vim version: VIM - Vi IMproved 8.2 (2019 Dec 12, compiled Jan 26 2020 11:45:38)
node version: v13.7.0
coc.nvim version: 0.0.74-98a709ec3f
term: dumb
platform: linux

## Messages
Messages maintainer: Bram Moolenaar <Bram@vim.org>
[coc.nvim] Update completed
```

### Vimrc (for both images)

  ```vim
  set nocompatible
  set runtimepath^=/home/user/.vim/pack/plugin/start/coc
  filetype plugin indent on
  syntax on
  set hidden
  ```

### Python test code (for both images)
```
def foo(aaa, bbb):
    pass

foo
```

### Issue

Open vim and `:CocInstall coc-python` then install the proposed linter.

When adding a left parenthesis after `foo` at the end of the Python file:
* Ubuntu : Nothing happen.
* ArchLinux : completion with `aaa` and `bbb` made possible.

### Ubuntu

![2020-02-11-080418_135x75_scrot](https://user-images.githubusercontent.com/33952527/74216609-3b835900-4ca5-11ea-9732-7f7f339a92a5.png)

### ArchLinux

![2020-02-11-074701_142x70_scrot](https://user-images.githubusercontent.com/33952527/74216613-40e0a380-4ca5-11ea-92a2-d6f2d20c31f4.png)

**Comment by fannheyward** [comment]
Post your `:CocOpenLog` please.

**Comment by pcoves** [comment]
# CocOpenLog

## Ubuntu

http://ix.io/2bnX

## ArchLinux

http://ix.io/2bnY

**Comment by pcoves** [comment]
Just realized I did not post the Dockerfiles I use to test in minimal conditions :+1: 

## Ubuntu

```
FROM ubuntu:18.04
RUN apt update && apt update -y
RUN apt install -y ctags git nodejs npm python3 python3-pip stow vim

RUN useradd -ms /bin/bash user
USER user
WORKDIR /home/user

RUN git clone https://github.com/neoclide/coc.nvim.git /home/user/.vim/pack/plugin/start/coc -b release
COPY ./vimrc /home/user/.vimrc

RUN pip3 install --user jedi pylint
```

## ArchLinux

```
FROM archlinux
RUN pacman --noconfirm -Syu ctags git nodejs npm python python-jedi python-pip stow vim

RUN useradd -ms /bin/bash user
USER user
WORKDIR /home/user

RUN git clone https://github.com/neoclide/coc.nvim.git /home/user/.vim/pack/plugin/start/coc -b release
COPY ./vimrc /home/user/.vimrc
```

**Comment by oblitum** [comment]
Vim popups were officially published for Vim only in 8.2 release. What you see for function signatures after `(` is signatureHelp, not simple pum menu completion, and it built on NeoVim floating windows or Vim popups.
